Update 2023_labscript_qc.md (#482)
unitaryfund/unitary.fund: src/content/grant/2023_labscript_qc.md

  - software
  - quantum
---

> To **Fred Jendrzejewski** to develop an SDK that allows cold atom research groups to make their experiments more accessible. The project has advanced quite substantially and the initial code has been merged with the **[sqooler](https://github.com/Alqor-UG/sqooler)** package such that theorists and experimentalists have a common framework. The system is now in a phase where it is looking for beta testers for the [hardware control](https://github.com/Alqor-UG/labscript-qc-example) as well as for the [simulator](https://github.com/Alqor-UG/sqooler-example) packages.

Reviewer: ```suggestion
To **Fred Jendrzejewski** to develop an SDK that allows cold atom research groups to make their experiments more accessible. The project has been merged with the **[sqooler](https://github.com/Alqor-UG/sqooler)** package such that theorists and experimentalists have a common framework. Beta testers are welcome to test examples for [hardware control](https://github.com/Alqor-UG/labscript-qc-example) and [simulators](https://github.com/Alqor-UG/sqooler-example).
```
